Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval/4.0.0/DataStandard_400_ApprovalTests.Verify.Extensions.SampleAlternativeEducationProgram.1.0.0_Std_4.0.0_Models_Entities_Entities.generated.approved.cs

```diff
@@ -407,6 +425,11 @@ public virtual ICollection<Entities.NHibernate.StudentAlternativeEducationProgra
                 // due to ServiceStack's lack of [OnDeserialized] attribute support.
                 // Back-reference is required by NHibernate for persistence.
                 // -------------------------------------------------------------
+                if (_studentAlternativeEducationProgramAssociationMeetingTimes is DeserializedPersistentGenericSet<Entities.NHibernate.StudentAlternativeEducationProgramAssociationAggregate.SampleAlternativeEducationProgram.StudentAlternativeEducationProgramAssociationMeetingTime> set)
+                {
+                    set.Reattach(this, "StudentAlternativeEducationProgramAssociationMeetingTimes");
```

Comment:
Reattach the collection on first access of the getter so that the containing entity can be reassociated with the NHibernate session without any database activity to synchronize state.